.github/workflows/deploy-pr-stack-and-run-int-tests.yml

```diff
@@ -74,3 +76,36 @@ jobs:
           role-to-assume: ${{ secrets.DEV_GH_ACTIONS_TOOLING_ROLE_ARN }}
       - name: Run integration tests
         run: npm run test:integration
+      - name: Deploy Jest HTML report to Github Pages
+        if: always()
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./reports
+          commit_message: 'Deploy Jest HTML report to gh-pages'
+          keep_files: true
+      - name: Check out gh-pages branch
+        if: always()
+        uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+      - name: Keep only last 5 jest-html reports in gh-pages
+        env:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config user.email "actions@github.com"
+          git config user.name "Github Actions"
+          git fetch
+          git branch
+          git checkout origin/main -- integration_tests/cleanup-jest-html-reports.sh
+          cd jest-html-reports
+          chmod +x ../integration_tests/cleanup-jest-html-reports.sh
```

Comment:
Should not be necessary, see other comment.

Reply:
removed as suggested